config/locales/publishers.yml

```diff
@@ -319,20 +319,27 @@ en:
       submit:
         cancel_caption: If you click cancel you will lose any unsaved information from this step. Completed details from previous steps will be saved in your draft jobs.
       summary:
-        create_another_job_listing: Create another job listing
-        date_expires: >-
-          The job listing will appear on the service until %{application_deadline}, after which it will no longer be
-          visible to jobseekers.
-        date_posted: Your job listing will be posted on %{date}.
-        date_posted_now: Your job listing has been posted.
-        feedback:
+        date_posted: Your job listing will be published on %{date}.
+        date_posted_now: Your job listing has been published.
+        feedback_link_html: <a href="%{url}" class="govuk-link">Give feedback about this service</a> (takes 30 seconds).
+        give_feedback: Give feedback
+        heading_published: Job listing published
```

Comment:
Perhaps having a `heading` key then subsequent `published` and `scheduled` keys would be better>